src/resmom_win/start_exec.c

```diff
@@ -153,10 +122,8 @@ static int direct_write_possible = 1;
 
 /* Local Varibles */
 
-#ifdef WIN32
 int		 	script_out;
 int			script_err;
-#endif
 
 static int	 script_in;	/* script file, will be stdin	  */
 static pid_t	 writerpid;	/* writer side of interactive job */
```

Comment:
Failure case in line 200. Logging error is missed?
(!exec_host_hookset && !exec_host2_hookset)) {
		return (1);

Reply:
Addressed in this commit : https://github.com/PBSPro/pbspro/pull/1737/commits/10a01689cdea949d5ac74258afb4e5ee8d90931f